Runtime/Native/Windows/NativeClient.cs

```diff
@@ -250,13 +250,24 @@ public void SetAttribute(string key, string value)
         public IEnumerator SendMinidumpOnStartup(ICollection<string> clientAttachments, IBacktraceApi backtraceApi)
         {
             // Path to the native crash directory
-            string nativeCrashesDir = Path.Combine(
-                    Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
-                    string.Format("Temp/{0}/{1}/crashes", Application.companyName, Application.productName));
+            string tmpDir = string.Format("Temp/{0}/{1}/crashes", Application.companyName, Application.productName);
+            string sdeckCrashPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), tmpDir);
+            string windowsCrashPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), tmpDir);
```

Comment:
```suggestion
            string tempDirectory = string.Format("Temp/{0}/{1}/crashes", Application.companyName, Application.productName);
            var crashDirectories = new string[2] { Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), tempDirectory), 
            Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), tempDirectory)
            };
            
            stirng nativeCrashesDir = "";
            for (var directory in crashDirectories) {
                 if (!Directory.Exists) {
                     continue;
                 }
                 nativeCrashesDir = directory;
                 break;
             }
             if (string.IsNullOrEmpty(nativeCrashesDir)) {
                 yield break;
             }
```

Reply:
This doesn't fully resolve an issue I noticed where the Deck will sometimes use the normal Windows path as opposed to the one that we found it uses.  Seems to crop up randomly.